src/common/options/global.yaml.in

```diff
@@ -104,8 +104,8 @@ options:
 - name: public_network_interface
   type: str
   level: advanced
-  desc: Interface name(s) from which to choose an address from a public_network to
-    bind to; public_network must also be specified.
+  desc: Interface name(s) from which to choose an address from a ``public_network`` to
+    bind to; ``public_network`` must also be specified.
```

Comment:
"bind to"?  I'm taking away your pedant card ;)

Reply:
I didn't write this.